Jenkinsfile

```diff
@@ -5,7 +5,7 @@ python_version = '3.6'
 env_vars = [
     "CRDS_SERVER_URL=https://jwst-crds.stsci.edu",
     "CRDS_PATH=./crds_cache",
-    "CRDS_CONTEXT=jwst_0508.pmap",
+    "CRDS_CONTEXT=jwst_0513.pmap",
```

Comment:
Same here.  This line can be deleted so that we just use the default.

Reply:
Done